articles/aks/confidential-containers-overview.md

```diff
@@ -67,14 +62,14 @@ It's important you understand the memory and processor resource allocation behav
 * CPU: The shim assigns one vCPU to the base OS inside the pod. If no resource `limits` are specified, the workloads don't have separate CPU shares assigned, the vCPU is then shared with that workload. If CPU limits are specified, CPU shares are explicitly allocated for workloads.
 * Memory: The Kata-CC handler uses 2 GB memory for the UVM OS and X MB additional memory where X is the resource `limits` if specified in the YAML manifest (resulting in a 2-GB VM when no limit is given, without implicit memory for containers). The [Kata][kata-technical-documentation] handler uses 256 MB base memory for the UVM OS and X MB additional memory when resource `limits` are specified in the YAML manifest. If limits are unspecified, an implicit limit of 1,792 MB is added resulting in a 2 GB VM and 1,792 MB implicit memory for containers.
 
-In this release, specifying resource requests in the pod manifests aren't supported. The Kata container ignores resource requests from pod YAML manifest, and as a result, containerd doesn't pass the requests to the shim. Use resource `limit` instead of resource `requests` to allocate memory or CPU resources for workloads or containers.
+In this release, specifying resource requests in the pod manifests isn't supported. containerd doesn't pass the requests to the Kata Shim, and as a result, reserving resources based on the pod manifest resource requests is not implemented. Use resource `limits` instead of resource `requests` to allocate memory or CPU resources for workloads or containers.
```

Comment:
In our previous meeting we had discussed that the user should be setting both `limits` and `requests` to the same value, because otherwise the scheduler will try to schedule the pod in a node that might not have enough memory. Could we also add this wording to this section?

Reply:
@Bickor, when you'll get a chance let's discuss together the memory management document we shared with your team while you were out on vacation last week. Based on that memory resource management model I don't think we'll have such a `limits` vs. `requests` problem anymore.